### PR TITLE
Update bestpractices doc README.md

### PR DESCRIPTION
**Pull request by lixuna**
Ref #330

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -8,9 +8,9 @@
 
 ## Introduction
 
-The goal of this group is to define a set of practices to evaluate the extent to which a CNF is [cloud native](https://github.com/cloud-native-principles/cloud-native-principles). CNFs can be evaluated against those best practices through test cases that are developed by other open source projects like the [CNF Test Suite](https://github.com/cncf/cnf-testsuite#cnf-conformance-test-suite).
+The goal of the Cloud Native Telecom Initiative (CNTI) Best Practices focus area is to define a set of practices to evaluate the extent to which a CNF is [cloud native](https://github.com/cloud-native-principles/cloud-native-principles). CNFs can be evaluated against those best practices through test cases that are developed by other open source projects like the [CNTI Test Catalog](https://github.com/cnti-testcatalog/testsuite).
```
Comment (jeffsaelens) [inline doc/README.md:11]: goal of -> focus area reads a little wonky

File: doc/README.md [hunk 2]
```diff
@@ -8,9 +8,9 @@
...
 
-CNF WG will rely on other industry initiatives (such as CNCF TUG) to come up with a definition of "cloud native" in relation to the telecom industry.
+The CNTI Best Practices focus group will rely on other industry initiatives (such as [NGMN](https://www.ngmn.org/highlight/ngmn-publishes-cloud-native-manifesto.html) to come up with a definition of "cloud native" in relation to the telecom industry.
```
Comment (jeffsaelens) [inline doc/README.md]: are we a focus group? once again, the wordiness of the line makes it difficult to consume. Not 100% sure how to condense this and still capture the point you were aiming for.

File: doc/README.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ As demonstrated by **Figure 1-2** below, When it comes to Cloud Native Network F
 
 ![Current Scope of CNF WG](./figures/cnf-wg-scope.png)
 
-The current focus of CNF WG is to define the best practices for the development and deployment of CNFs. Other areas are left for future focus.
+The current focus of CNTI Best Practices Focus Area is to define the best practices for the development and deployment of CNFs. Other areas are left for future focus.
```
Comment (jeffsaelens) [inline doc/README.md]:
```suggestion
The current goal of CNTI Best Practices Focus Area is to define the best practices for the development and deployment of CNFs. Other areas are left for future focus.
```

File: doc/cbpps/0001-cnf-best-practice-proposal-process.md [hunk 4]
```diff
@@ -62,7 +62,7 @@ Without a standardized mechanism for describing important Cloud Native best prac
 
 The purpose of the CBPP process is to reduce the amount of "tribal knowledge" in our community. By moving decisions from a smattering of mailing lists, video calls, and hallway conversations into a well tracked artifact, this process aims to enhance communication and discoverability.
 
-An important goal of the CBPP process is ensuring that the process for submitting the content is both clear and efficient. The CBPP process is intended to create high-quality, uniform design documents for the CNF WG to deliberate.
+An important goal of the CBPP process is ensuring that the process for submitting the content is both clear and efficient. The CBPP process is intended to create high-quality, uniform design documents for the CNTI Best Practices focus area to deliberate.
```
Comment (jeffsaelens) [inline doc/cbpps/0001-cnf-best-practice-proposal-process.md:65]: focus group?

File: doc/cbpps/README.md [hunk 5]
```diff
@@ -10,7 +10,7 @@ Follow the process outlined in the [CBPP process document](cnf_best_practice_pro
 
 ## CNF Use Cases
 
-CBPP shall always refer to at least one of the CNF WG use cases. Those use cases serve as the practical and concrete basis for deriving, discussing and evaluating the best practices for CNFs. Their value is in providing context and a reality check for CBPPs. With help of use cases we make sure that CBPPs are addressing a specific problem or need from the real world. Please use the [Use Case Template](../use-case/NNNN-UC-template.md) to contribute a use case.
+CBPP shall always refer to at least one of the CNF use cases. Those use cases serve as the practical and concrete basis for deriving, discussing and evaluating the best practices for CNFs. Their value is in providing context and a reality check for CBPPs. With help of use cases we make sure that CBPPs are addressing a specific problem or need from the real world. Please use the [Use Case Template](../use-case/NNNN-UC-template.md) to contribute a use case.
```
Comment (jeffsaelens) [inline doc/cbpps/README.md]:
```suggestion
CBPP shall always refer to at least one CNF use case. These use cases serve as the practical and concrete basis for deriving, discussing and evaluating the best practices for CNFs. Their value is in providing context and a reality check for CBPPs. With help of use cases we make sure that CBPPs are addressing a specific problem or need from the real world. Please use the [Use Case Template](../use-case/NNNN-UC-template.md) to contribute a use case.
```

**Review by jeffsaelens — CHANGES_REQUESTED** [review]
Focus group, Best Practices focus area, etc... are all pretty cumbersome from a mouthful perspective. Is this standard wording for LFN or could we look for potential options to streamline these phrases a smidge?

Maybe CNTI Community vs focus group as an example?

File: doc/README.md (hunk 2 above)
Comment (taylor) [reply doc/README.md]: @jeffsaelens the focus group or focus area are words that do not have baggage in LF(N)/CNCF like project and initiative. We have the overall effort (CNTi) and the sub-thing or focus area for best practices (previously called CNF WG)

Comment (taylor) [reply doc/README.md]: @lixuna Maybe this should say focus area instead of focus group

Comment (taylor) [inline doc/README.md]:
```suggestion
The CNTI Best Practices focus area will rely on other industry initiatives (such as [NGMN](https://www.ngmn.org/highlight/ngmn-publishes-cloud-native-manifesto.html) to come up with a definition of "cloud native" in relation to the telecom industry.
```

File: doc/README.md (hunk 1 above)
Comment (taylor) [reply doc/README.md:11]: @jeffsaelens If you don't have a grammar alternative suggestion we will leave it like this for now.

File: doc/README.md (hunk 2 above)
Comment (lixuna) [inline doc/README.md]:
```suggestion
The CNTI Best Practices focus area will rely on other industry initiatives (such as [NGMN](https://www.ngmn.org/highlight/ngmn-publishes-cloud-native-manifesto.html) to come up with a definition of "cloud native" in relation to the telecom industry.
```

**Pull request merged**